tests/test_plumbing.py

```diff
@@ -398,6 +398,30 @@ def test_choicefield_choices_enum():
     assert schema['enum'] == ['bluepill', 'redpill', '', None]
     assert 'type' not in schema
 
+    schema = build_choice_field(serializers.ChoiceField(
+        choices=[1, 2], allow_blank=True
+    ))
+    assert schema['enum'] == [1, 2, '']
+    assert 'type' not in schema
```

Comment:
Note that this is a change in behavior.

Previously this generated `{"type": "integer"}` because all `choices=` arguments are int and `allow_blank` wasn't considered.

I think this is more correct, but I'm open to restoring the old behavior for this.

Reply:
According to https://swagger.io/docs/specification/data-models/data-types/#mixed-type, `type` is expected to be a single value so this looks more correct to me too.
Would be nice to support `oneOf` someday that would help in this case to have:
```yaml
oneOf:
  - int
  - string
```
But that's out-of-scope for this PR.